src/components/Records/Shared/KeywordTooltip.vue

```diff
@@ -7,12 +7,16 @@
       <template #activator="{ on, attrs }">
         <div
           v-bind="attrs"
+          class="ellipse-width-80 text-center"
           v-on="on"
         >
           {{ keyword.label }}
         </div>
       </template>
       <div class="tooltip">
+        <div :class="getChipColor(keyword)">
```

Comment:
you need to add a condition to make sure the type is present of you need to modify the edit general information panel to also pass the type.
Otherwise, we get something weird:
![image](https://user-images.githubusercontent.com/6538906/104933724-1cce1d80-595e-11eb-8ea8-ea1282ebeec5.png)

Reply:
thanks for spotting this bug. I ll check that